gulpfile.js

```diff
@@ -193,14 +193,6 @@ gulp.task("watch", ["serve"], function () {
     );
     gulp.watch(
         [
-            path.join(DOCS_DIR, "**", "*.md"),
-            path.join(DOCS_DIR, "**", "*.html"),
-        ],
-        {interval: WATCH_INTERVAL},
-        ["configs", "data"]
```

Comment:
Just making sure: this is to disable regeneration of toc/version data and configs while watching, correct?